src/app/kontakt/components/InfoItem.tsx

```diff
@@ -0,0 +1,17 @@
+import { ReactNode } from "react";
+
+interface InfoItemProps {
+  icon: ReactNode;
+  label: string;
+  content: ReactNode;
+}
+
+export const InfoItem = ({ icon, label, content }: InfoItemProps) => (
+  <div className="flex items-center gap-3 desktop:mb-8 desktop:gap-8">
+    <div className="h-8 w-8 flex-shrink-0">{icon}</div>
+    <div>
+      <p className="desktop:mb-4">{label}</p>
+      <div className="font-medium">{content}</div>
```

Comment:
I like this component. Quite clean. I think it would be even better if we would have `content` accepted as just `children`. Would be simpler 😄